keadm/cmd/keadm/app/cmd/helm/installer.go

```diff
@@ -277,7 +277,10 @@ func (cu *KubeCloudHelmInstTool) runHelmManifest(r *Renderer, stdout io.Writer)
 	}
 	buf.WriteString(manifests)
```

Comment:
should we handle the err return by buf.WriteString?

Reply:
`func (b *Buffer) WriteString(s string) (n int, err error) {
	b.lastRead = opInvalid
	m, ok := b.tryGrowByReslice(len(s))
	if !ok {
		m = b.grow(len(s))
	}
	return copy(b.buf[m:], s), nil
}`

// WriteString appends the contents of s to the buffer, growing the buffer as
// needed. The return value n is the length of s; err is always nil. If the
// buffer becomes too large, WriteString will panic with ErrTooLarge.
I think we do not need handle